extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,228 @@
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>The baseColor of the material (yellow, in this example) is used to tint the light being transmitted.</em></figcaption>
+</figure>
+
+## Refraction
+Since the surface is considered to be infinitely thin, we will ignore macroscopic refraction caused by the orientation of the surface. However, microfacets on either side of the thin surface will cause light to be refracted in random directions, effectively blurring the transmitted light. That is, the roughness of the surface directly causes the transmitted light to become blurred. This microfacet lobe is exactly the same as the specular lobe except sampled along the line of sight through the surface.
+<figure>
+  <img src="./figures/Roughness.png"/>
+<figcaption><em>Refraction due to surface roughness.</em></figcaption>
+</figure>
+
+## Blend Mode
+The glTF `blendMode` is used for alpha-as-coverage, NOT for physically-based transparency (i.e. this extension). If alpha-as-coverage is not being used, the blend mode of the material should be set to "OPAQUE" even though it is transparent. Again, it's helpful to think of alpha-as-coverage as whether the physical surface is there or not. `transmission` applies to the surface material that exists.
+Note that alpha-as-coverage can still be used along with transmission as shown in the example below.
+<figure>
+  <img src="./figures/TransmissionWithMask.png"/>
+<figcaption><em>Alpha coverage and optical transparency can be used at the same time so that some areas of a surface are transparent while others disappear entirely.</em></figcaption>
+</figure>
+
+## Transparent Metals
+Metals effectively absorb all refracted light (light that isn't reflected), preventing transmission.
+The metallic parameter of a glTF material effectively scales the `baseColor` of the material toward black while, at the same time scaling the F0 (reflectivity) value towards 1.0. This makes the material opaque for metallic values of 1.0 because transmitted light is attenuated out by absorption. Therefore, for a material with `metallicFactor=1.0`, the value of `transmissionFactor` doesn't matter.
+
+## Transmission BTDF ##
+From the core [glTF BRDF](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#appendix-b-brdf-implementation), we have:
+
+*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub>
+
+*f*<sub>*diffuse*</sub> = (1 - *F*) * *diffuse*
+
+*f*<sub>*specular*</sub> = *F* * *G* * *D* / (4 * dot(*N*, *L*) * dot(*N*, *V*)), where *F* is the Surface Reflection Ratio.
+
+We will now add an additional term for the transmitted light:
+
+*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub> + *f*<sub>*transmission*</sub>
+
+*f*<sub>*transmission*</sub> = (1 - *F*) * *T* * *baseColor* * *D<sub>T</sub>* * *D<sub>G</sub>* / *(4 * abs(dot(N, L))* * *abs(dot(N, V)))*
```

Comment:
Minor, but I think you meant G<sub>T</sub>.  May want to also mention something about G<sub>T</sub> below.  There's a related conversation about this term on the [KHR_materials_volume PR](https://github.com/KhronosGroup/glTF/pull/1726#discussion_r465386060)